plugin/ochttp/stats.go

```diff
@@ -60,8 +68,16 @@ var (
 	DefaultLatencyDistribution = view.Distribution(0, 1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20, 25, 30, 40, 50, 65, 80, 100, 130, 160, 200, 250, 300, 400, 500, 650, 800, 1000, 2000, 5000, 10000, 20000, 50000, 100000)
 )
 
-// Package ochttp provides some convenience views.
-// You need to subscribe to the views for data to actually be collected.
+// Convenience HTTP client views.
+//
+// In many cases, you will want to create your own views over the measures
+// provided by this package, but for simple programs these convenience views
```

Comment:
"but for simple programs these convenience views"

This is misleading. We think these views are essentially important and they are not just for simple programs.